scripts/prepare_ports.py

```diff
@@ -20,8 +20,7 @@ def download_ports():
     The following csv file was downloaded from the webpage https://msi.nga.mil/Publications/WPI as a csv file that is updated monthly as mentioned on the webpage. The dataset contains 3711 ports.
     """
     fn = "https://msi.nga.mil/api/publications/download?type=view&key=16920959/SFH00000/UpdatedPub150.csv"
-    storage_options = {"User-Agent": "Mozilla/5.0"}
-    wpi_csv = pd.read_csv(fn, index_col=0, storage_options=storage_options)
+    wpi_csv = pd.read_csv(fn, index_col=0)
```

Comment:
can you explain why `storage_options` is deleted and what the purpose was?

Reply:
@energyLS When downloading csv files with pd.read_csv, the storage_options are transferred as header in the request. They are there to make the request look more "credible", in this case to indicate that the request is being sent from a Mozilla Firefox browser. For some reason, however, this is exactly what triggers this HTTP error 403. As soon as we do not send this header, we get the data packet returned. :)

Reply:
Thanks, great catch. Merging!